gapis/gfxapi/api.go

```diff
@@ -36,8 +36,8 @@ type API interface {
 	// ConstantSets returns the constant set pack for the API.
 	ConstantSets() *constset.Pack
 
-	// GetFramebufferAttachmentInfo returns the width, height and format of the specified framebuffer attachment.
-	GetFramebufferAttachmentInfo(state *State, attachment FramebufferAttachment) (width, height uint32, format *image.Format, err error)
+	// GetFramebufferAttachmentInfo returns the width, height, revised index, and format of the specified framebuffer attachment.
```

Comment:
I updated the comment but basically it is an API-specific representation of the same attachment.

This comes up specifically as a problem for subcommands. 
When walking all of the framebufferChanges, we call GetFramebufferAttachmentInfo after every subcommand. (i.e. in a callback that gets called after each subcommand). This will map the given "FramebufferAttachment" to the index in the Vulkan "VkFramebuffer" object using whatever RenderPass is currently active during that subcommand.

If there are multiple RenderPasses, then this might be different at the end of the VkQueueSubmit atom, and when we actually have to go and do the replay, we need to know what the right index is before we mutate that atom. (Since we will have to modify it).